datafusion/src/sql/parser.rs

```diff
@@ -192,6 +194,35 @@ impl<'a> DFParser<'a> {
         }
     }
 
+    fn parse_partitions(&mut self) -> Result<Vec<String>, ParserError> {
```

Comment:
similar to `parse_columns`. I didn't reuse `parse_columns` because partitions only have identifier/name, there is no type or other options like column

https://github.com/apache/arrow-datafusion/blob/f50e8000ee4de8432d7b59b63c18a8e0171a072b/datafusion/src/sql/parser.rs#L227-L229

Reply:
sorry missed this comment yesterday 👍